### DISCUSSION TITLE
Lost chia in change for paying fee

### DISCUSSION THREAD

**Discussion by Harryols** [Chia-Network/chia-blockchain#10465] [Support]
Hi,
I have lost around 2 chia because something went wrong, I think it's the change for paying some fees that is lost. My wallet is fully synced and port 8444 open. I have some transaction i suspect could be the issue.

**Comment by Harryols** [comment]
![Screenshot_20220227-182706~2.png](https://user-images.githubusercontent.com/4333129/155892841-92ce4042-3715-4157-a8b3-31344de20ebd.png)

**Comment by loppefaaret** [comment]
If you try and install 1.3.0 release, do you still have the same problem ?

**Reply by Harryols** [reply]
I still have 0.7 XCH missing in the new stable release you mention.

**Reply by loppefaaret** [reply]
in that case, i have to ask that you manually reset your wallet database, it is located under your users home folder, in `~\.chia\mainnet\wallet\db\` and is containing the same fingerprint as you see in the GUI when selecting it.
First close down the Chia client, then remove the `.sqlite` file, with the filename corresponding to the key - then re-open the Chia client, and let the wallet sync back up from 0 - this should allow it to get all the transaction in the correct order, and show the full balance as intended.

**Reply by Harryols** [reply]
Deleting wallet db worked - thanks
